ale/MSD.py

```diff
@@ -36,17 +36,18 @@ def plotter():
     return plotter
 
 def self_diffusion_coefficient(atom_list) :
-    """The self_diffusion_coefficient(t, atom_list) function calculates and returns the
+    """The self_diffusion_coefficient(atom_list) function calculates and returns the
     self diffusion coefficient. The function takes an atom_list at different time steps
     which it sends to the MSD(t,atom_list) function to retrieve the MSD.
     The lindemann_critertion() first checks if the element is a solid or liquid. For
     solids we approximate the self_diffusion_coefficient as 0 and for liquids the self
-    diffusion coefficient is taken as the slope of the mean-square-displacement."""
+    diffusion coefficient is taken as the slope of the mean-square-displacement.
+    Self diffusion coefficient is returned in units m^2/s."""
     time_step = len(atom_list) - 1 #Take the system at the last accessible time
-    t = time_step * 5 * 1E-15
-    MSD_meter = 1E-10 * MSD(time_step, atom_list) #Convert from Å to meter TODO: Angstrom^2????
+    t = time_step * 5 * 1E-15 #Convert time_step to seconds
```

Comment:
The timestep or dt that is going in the MSD function is the dt that the user chooses in the config file. I think it might create trouble if that is not the same dt. So maybe we should make it an in-variable in this function as well? Check how it is done in the MSD function.